src/pudl/metadata/resources/gridpathratoolkit.py

```diff
@@ -0,0 +1,56 @@
+"""Table definitions for data coming from the GridPath Resource Adequacy Toolkit."""
+
+from typing import Any
+
+RESOURCE_METADATA: dict[str, dict[str, Any]] = {
+    "out_gridpathratoolkit__hourly_available_capacity_factor": {
+        "description": (
+            "Hourly capacity factors defining the capacity available from an "
+            "aggregated group of generators, stated as a fraction of the aggregate "
+            "nameplate capacity of the group. "
+            "This table contains a mix of profiles representing whole regions and "
+            "individual plants, where the individual plants are hybrid wind or solar "
+            "plus electricity storage facilities. For the hybrid facilities the "
+            "capacity factor represents the available output of only the renewable "
+            "generators. Estimates of the generation profiles have been extended "
+            "across a longer range of dates than the original data. For wind, the "
+            "years 2015-2020 are synthesized data and for solar 2020 is synthesized. "
+            "See the Appendix of the GridPath Resource Adequacy Toolkit report for "
+            "details."
+        ),
+        "schema": {
+            "fields": [
+                "datetime_utc",
+                "aggregation_group",
+                "capacity_factor",
+            ],
+            "primary_key": ["datetime_utc", "aggregation_group"],
+        },
+        "sources": ["gridpathratoolkit"],
+        "field_namespace": "gridpathratoolkit",
+        "etl_group": "gridpathratoolkit",
+    },
+    "core_gridpathratoolkit__assn_generator_aggregation_group": {
+        "description": (
+            "This table defines which individual generator profiles are combined when "
+            "creating aggregated capacity factors / generation profiles. Generator "
+            "capacity is used to weight the contribution of each generator in the "
+            "resulting aggregated profiles, and is made available in this table for "
+            "convenience and legibility. The resulting aggregated profiles are stored "
+            "in :ref:`out_gridpathratoolkit__hourly_available_capacity_factor`."
```

Comment:
does :ref:`table_name` make sense in all of the contexts of where this table description is going to go? i don't think we are doing this anywhere else

Reply:
It'll show up as a link in the data dictionary, but not in the Datasette metadata. It looks like we've got a couple of references like this in the MCOE tables.  One advantage of making it a ref is that if the table name changes, the docs build will fail, and we'll be reminded to update the reference. But yes it will look a bit wonky in the datasette metadata.

Reply:
okay yea thats a little weird in non-docs contexts but alas i do like that the docs build would fail :-/